### PR TITLE
Renamed bridgeId to bridgeCallData

### PR DESCRIPTION
**Pull request by benesjan**
# Description

Renamed bridgeId to bridgeCallData.

# Checklist:

- [x] I have reviewed my diff in github, line by line.
- [x] Every change is related to the PR description.
- [x] There are no unexpected formatting changes, or superfluous debug logs.
- [x] The branch has been rebased against the head of its merge target.
- [x] I'm happy for the PR to be merged at the reviewers next convenience.

### CODE REVIEW

File: src/specs/TEMPLATE.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You can view an example spec [here](#add-link).
    - For each interaction define:
      - All input tokens, including [AztecType](https://github.com/AztecProtocol/aztec-connect-bridges/blob/master/src/aztec/libraries/AztecTypes.sol) info
      - All output tokens, including [AztecType](https://github.com/AztecProtocol/aztec-connect-bridges/blob/master/src/aztec/libraries/AztecTypes.sol) info
-     - All relevant bridge ids (may fill in after deployment)
```
Comment (LHerskind) [inline src/specs/TEMPLATE.md:22]: SHould be `BridgeAddressId`

File: src/test/aztec/base/BridgeTestBase.sol [hunk 2]
```diff
@@ -23,9 +23,9 @@ abstract contract BridgeTestBase is Test {
     error INVALID_ASSET_ADDRESS();
     error INVALID_LINKED_TOKEN_ADDRESS();
     error INVALID_LINKED_BRIDGE_ADDRESS();
-    error INVALID_BRIDGE_ID();
+    error INVALID_BRIDGE_CALL_DATA();
     error INVALID_BRIDGE_ADDRESS();
-    error BRIDGE_ID_IS_INCONSISTENT();
```
Comment (LHerskind) [inline src/test/aztec/base/BridgeTestBase.sol:28]: Not sure you want to change these to something else than the deployed version. If getting the `INVALID_BRIDGE_ID()` error, it won't be able to match with an error from here, so you might have issues debugging it.

**Review by LHerskind — CHANGES_REQUESTED** [review]
Minor comments

File: src/specs/TEMPLATE.md (hunk 1 above)
Comment (benesjan) [reply src/specs/TEMPLATE.md:22]: Addressed in 3441df17d0a6c77fe9510de3014ba8a985689d9f

File: src/test/aztec/base/BridgeTestBase.sol (hunk 2 above)
Comment (benesjan) [reply src/test/aztec/base/BridgeTestBase.sol:28]: Ok, I will leave it there for now and add there a TODO.

Comment (benesjan) [reply src/test/aztec/base/BridgeTestBase.sol:28]: Addressed in a5ac4a5e5c924e726777869d08320eed13b5c9fa

**Review by LHerskind — APPROVED** [review]

**Pull request merged**